

From: Michael Ekstrand <mdekstrand@drexel.edu>


---
 Cargo.toml |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 712bdae8b..3003db59f 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -243,6 +243,9 @@ winapi = "=0.3.9"
 windows-sys = { version = "0.59.0", features = ["Win32_Foundation", "Win32_Media", "Win32_Storage_FileSystem", "Win32_System_IO", "Win32_System_WindowsProgramming", "Wdk", "Wdk_System", "Wdk_System_SystemInformation", "Win32_Security", "Win32_System_Pipes", "Wdk_Storage_FileSystem", "Win32_System_Registry", "Win32_System_Kernel", "Win32_System_Threading", "Win32_UI", "Win32_UI_Shell"] }
 winres = "=0.1.12"
 
+[patch.crates-io]
+libffi-sys = { git = "https://github.com/tov/libffi-rs", branch = "master" }
+
 [profile.release]
 codegen-units = 1
 incremental = true
