drop zlib

From: Michael Ekstrand <mdekstrand@drexel.edu>


---
 Cargo.toml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 8b4319612..594daaa41 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -95,7 +95,7 @@ encoding_rs = "=0.8.33"
 ecb = "=0.1.2"
 fastwebsockets = "=0.5.0"
 filetime = "0.2.16"
-flate2 = { version = "1.0.26", features = ["zlib-ng"], default-features = false }
+flate2 = { version = "1.0.26" }
 fs3 = "0.5.0"
 futures = "0.3.21"
 glob = "0.3.1"
